derp/derp_server.go

```diff
@@ -508,6 +540,18 @@ func (s *Server) accept(nc Conn, brw *bufio.ReadWriter, remoteAddr string, connN
 	return c.run(ctx)
 }
 
+// toKB converts bytes to kilobytes, rounding up if necessary.
```

Comment:
why are kilobytes the unit used everywhere?

Reply:
eh just an arbitrary choice, will switch to bytes